<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>937</width>
    <height>565</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>937</width>
    <height>565</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>937</width>
    <height>565</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>管理计划</string>
  </property>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>921</width>
     <height>551</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout" stretch="5,2">
    <item>
     <widget class="QTableWidget" name="PlanList">
      <property name="maximumSize">
       <size>
        <width>552</width>
        <height>492</height>
       </size>
      </property>
      <column>
       <property name="text">
        <string>检测计划</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>检测进度</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>检测时间</string>
       </property>
      </column>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0,0,0,0">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="label">
          <property name="maximumSize">
           <size>
            <width>70</width>
            <height>19</height>
           </size>
          </property>
          <property name="text">
           <string>计划名：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="planname_le">
          <property name="maximumSize">
           <size>
            <width>210</width>
            <height>34</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QPushButton" name="newplan_btn">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="maximumSize">
           <size>
            <width>203</width>
            <height>63</height>
           </size>
          </property>
          <property name="text">
           <string>新建计划</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QPushButton" name="startplan_btn">
          <property name="maximumSize">
           <size>
            <width>203</width>
            <height>63</height>
           </size>
          </property>
          <property name="text">
           <string>启动计划</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QPushButton" name="changeplan_btn">
          <property name="maximumSize">
           <size>
            <width>203</width>
            <height>63</height>
           </size>
          </property>
          <property name="text">
           <string>修改计划</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_7">
        <item>
         <widget class="QPushButton" name="delateplan_btn">
          <property name="maximumSize">
           <size>
            <width>203</width>
            <height>63</height>
           </size>
          </property>
          <property name="text">
           <string>删除计划</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
